index.html

```diff
@@ -120,7 +130,128 @@ <h2>Introduction</h2>
       <p class="issue" data-number="7"></p>
     </section>
 
-  <section id="conformance"></section>
+    <section id="conventions">
+      <h2>Documentation Conventions</h2>
+      <p>
+        This document uses the same conventions as [[TTML2]] for the specification of parameter attributes, styling attributes and metadata elements. In particular:
+      </p>
+      <p>
+        Section 2.3 of [[TTML2]] specifies conventions used in the [[XML]] representation of elements; and
+        Sections 6.2 and 8.2 of [[TTML2]] specify conventions used when specifying the syntax of attribute values.
+      </p>
+      <p>
+        All content of this specification that is not explicitly marked as non-normative is considered to be normative.
+        If a section or appendix header contains the expression "non-normative", then the entirety of the section or appendix is considered non-normative.
+      </p>
+      <p>
+        This specification uses Feature designations as defined in Appendices E at [[TTML2]]:
+        when making reference to content conformance, these designations refer to the syntactic expression or the semantic capability associated with each designated Feature; and
+        when making reference to processor conformance, these designations refer to processing requirements associated with each designated Feature.
+        If the name of an element referenced in this specification is not namespace qualified, then the TT namespace applies (see <a href="#namespaces">9.3 Namespaces</a>.)
+      </p>
+    </section>
+
+    <section id="definitions">
+      <h2>Definitions</h2>
+      <p>
+        The following terms are used in this proposal:
+      </p>
+      <p>
+        <dfn>Audio description</dfn>	An audio rendition of a <a>Description</a> or a set of <a>Descriptions</a>.
+      </p>
+      <p>
+        <dfn data-lt="Description|Descriptions">Description</dfn>  A set of words that describe an aspect of the programme presentation, suitable for rendering into audio by means of vocalisation and recording or used as a <a>Text Alternative</a> source for text to speech translation.
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-default-region">Default Region</dfn> See Section 11.3.1.1 at [][TTML2]].
+      </p>
```

Comment:
This definition is not used, as expected since we don't want to specify layout. Can we remove it?

Reply:
Yes